src/core/client_channel/backup_poller.cc

```diff
@@ -63,9 +63,13 @@ static bool g_backup_polling_disabled;
 
 void grpc_client_channel_global_init_backup_polling() {
   // Disable backup polling if EventEngine is used everywhere.
+#ifdef GRPC_DO_NOT_INSTANTIATE_POSIX_POLLER
+  g_backup_polling_disabled = false;
```

Comment:
note that we also have the experiment `event_engine_callback_cq` which was added in 1.69. If that is enabled, grpc_iomgr_run_in_background will return true, and the backup poller will not be started. Why not add this logic in grpc_client_channel_start_backup_polling/grpc_client_channel_stop_backup_polling instead?

Reply:
Done. If `GRPC_DO_NOT_INSTANTIATE_POSIX_POLLER` is defined, the all of the logic to disable the backup poller is removed at compile time.